apps/builder/app/shared/context.server.ts

```diff
@@ -2,6 +2,7 @@ import type { AppContext } from "@webstudio-is/trpc-interface/index.server";
 import env from "~/env/env.server";
 import { authenticator } from "~/services/auth.server";
 import { trpcClient } from "~/services/trpc.server";
+import { entryApi } from "./entri/entri-api.server";
```

Comment:
I thought we agreed that all entri stuff would be in one folder in one package, but it seems we now have a bunch of entri folders in various places

Reply:
Everything is in a single folder entri, whats wrong?